easybuild/easyconfigs/s/scanpy/scanpy-1.7.2-foss-2020b.eb

```diff
@@ -0,0 +1,104 @@
+easyblock = 'PythonBundle'
+
+name = 'scanpy'
+version = '1.7.2'
+
+homepage = 'https://scanpy.readthedocs.io/en/stable/'
+description = """Scanpy is a scalable toolkit for analyzing single-cell gene expression data built
+ jointly with anndata. It includes preprocessing, visualization, clustering, trajectory inference
+ and differential expression testing. The Python-based implementation efficiently deals with
+ datasets of more than one million cells.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'openmp': True}
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
+    ('Autotools', '20200321'),
+    ('flex', '2.6.4'),
+    ('Bison', '3.7.1'),
+    ('CMake', '3.18.4'),
+]
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('h5py', '3.1.0'),
+    ('matplotlib', '3.3.3'),
+    ('networkx', '2.5'),
+    ('numba', '0.52.0'),
+    ('pandas', '1.2.4'),
+    ('PyTables', '3.6.1'),
+    ('statsmodels', '0.12.1'),
+    ('scikit-learn', '0.23.2'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('tqdm', '4.60.0', {
```

Comment:
We have an easyconfig for `tqdm` which you can leverage via `dependencies`, that seems better than installing it again as extension?